src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/Link.js

```diff
@@ -39,6 +42,60 @@ class Link extends Component<Props> {
     @observable overlayTitle: ?string;
     @observable overlayTarget: ?string = DEFAULT_TARGET;
     @observable overlayAnchor: ?string;
+    @observable displayProperties: Object = {};
+    @observable displayPropertiesLoading: boolean = false;
+
+    constructor(props: Props) {
+        super(props);
+
+        this.load(this.props.value);
+    }
+
+    componentDidUpdate(prevProps: Props) {
+        if (prevProps.value !== this.props.value) {
+            this.load(this.props.value);
+        }
+    }
+
+    @action load = (value) => {
```

Comment:
This is a bugfix which should target the `2.4` branch. Would also like that @niklasnatter have a look at this.